src/objects/Clickable.js

```diff
@@ -31,6 +31,20 @@ function click() {
   this.destroy();
   this._game.add.existing(new Toast(this._game, 'You\'ve acquired ' + this.name + '!', 5));
   this._game.player.inventory.push(this.id);
+  var items = this._game.room.area.items;
+  var remove_item = -1;
+  //because it is checking the id of the item and not the actual item
+  for (let i = 0; i < items.length; i++) {
+    if (items[i].id === this.id){
+      remove_item = i;
+    }
+  }
+  if (remove_item > -1) {
+    items.splice(remove_item, 1);
+  } else {
+    console.log('unexpected error, please check');
```

Comment:
If this is meant to be more of an assert function, I think you can try using [console.assert()](https://developer.mozilla.org/en-US/docs/Web/API/console/assert).